pontoon/base/static/css/style.css

```diff
@@ -489,34 +498,49 @@ label {
   width: 219px; /* Instead of 220px; because of padding */
 }
 
-.locale.select {
-  padding-left: 0;
-  width: 100%;
+.part .menu nav ul li {
+  border: 1px solid transparent;
+  border-bottom-color: #5E6475;
 }
 
-.locale.select .menu {
-  bottom: auto;
-  display: inline-block;
-  position: relative;
-  width: 100%;
+.part .menu nav ul li.hover {
+  background: transparent;
+}
 
-  -moz-box-sizing: border-box;
-  box-sizing: border-box;
+.part .menu nav ul li.active {
```

Comment:
I asked craigcook and he recommended, among other things https://smacss.com/. I'll be checking it out to see how I feel about it. :D